templates/web/config.j2

```diff
@@ -0,0 +1,32 @@
+; https://github.com/CPAN-API/metacpan-conf-private/blob/master/metacpan-web/metacpan_web_local.conf
+;
+; do not use localhost:5000 for the production server since Starman seems to have
+; problems with closing connections thus causing zombie www processes.
+; use the nginx vhost instead since nginx is much better at handling connections.
```

Comment:
Are we still using `nginx` with `traefik`?

Reply:
This was a carry over from the original configuration file. Removing.